src/alias.rs

```diff
@@ -0,0 +1,13 @@
+use common::*;
+
+pub struct Alias<'a> {
+  pub name: &'a str,
+  pub target: &'a str,
+  pub line_number: usize,
+}
+
+impl<'a> Display for Alias<'a> {
+  fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    writeln!(f, "alias `{}` = `{}`", self.name, self.target)
```

Comment:
I don't think this is being used, but let's keep it, and use it in dump (I'll add a comment there too). The backticks should also be removed, because `--dump` should produce a parsable justfile. (It doesn't at the moment, but i have a diff on the way which will fix that.)

Reply:
Also, it should be `write!` instead of `writeln!`, since conventionally the `Display` implementation doesn't add its own newline, leaving it up to the caller instead.